packages/backend/src/payment-method/ilp/connector/core/test/telemetry.test.ts

```diff
@@ -0,0 +1,125 @@
+import { TelemetryService } from '../../../../../telemetry/service'
+import { initIocContainer } from '../../../../../index'
+import { Config } from '../../../../../config/app'
+import { IocContract } from '@adonisjs/fold'
+import { AppServices } from '../../../../../app'
+import {
+  incrementPreparePacketCount,
+  incrementFulfillOrRejectPacketCount,
+  incrementAmount
+} from '../telemetry'
+import { IlpResponse } from '../middleware/ilp-packet'
+import { IlpFulfillFactory, IlpRejectFactory } from '../factories'
+import { ValueType } from '@opentelemetry/api'
+
+describe('Connector Core Telemetry', () => {
```

Comment:
we should add a few tests for handling non-positive cases (e.g., "... doesn't collect metrics for unfulfillable packets")

Reply:
So should I just pass in `unfulfillable` as `true` and check it doesn't call incrementCounter?

Reply:
I did my best, have a look and see if the new tests meet this criteria :)